pkg/sdk/file_format.go

```diff
@@ -572,7 +572,7 @@ func (opts *FileFormatTypeOptions) validate() error {
 		return fmt.Errorf("IgnoreUTF8Errors and ReplaceInvalidCharacters cannot be set simultaneously")
 	}
 
-	validEnclosedBy := []string{"null", "'", `"`}
+	validEnclosedBy := []string{"NONE", "'", `"`}
```

Comment:
It would be nice to see this value used in one of the SDK integration tests (`file_format_integration_test.go`).

Reply:
added integration test